src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,58 @@
+.icon {
+  display: block;
+
+  width: 100%;
+  padding-bottom: 100%;
+
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+
+  &--menu {
+    padding-bottom: 80%;
+
+    background-image: url(../images/menu.svg);
+
+    &:hover {
+      background-image: url(../images/menu_hover.svg);
+
+      transition: transform 0.5s;
```

Comment:
don't add transition on hover, add it under the general style because now it's not smooth when I remove my cursor from it

https://user-images.githubusercontent.com/106310796/197961579-4925b23a-5a91-4a14-8ed1-de020a05e029.mov